cel/README.md

```diff
@@ -0,0 +1,221 @@
+
+# Common Expression Language
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubernetes/experimental/blob/master/LICENSE)
+
+This is an experimental project that provides support for Common Expression Language (CEL) in Tekton Pipelines.
+The functionality is provided by a controller that implements the Custom Task interface. Its use cases include 
+evaluating complex expressions to be used in [`WhenExpressions`](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#guard-task-execution-using-whenexpressions)
+in subsequent `Tasks` to guard their execution. 
+
+- [Install](#install)
+- [Usage](#usage)
+- [Uninstall](#uninstall)
+
+## Install
+
+Install and configure `ko`
+
+```
+ko apply -f celcontroller.yaml
+```
+
+This will build and install the controller on your cluster, in the namespace `cel`
```

Comment:
nit: Can this be `tekton-cel-task` or something? There might be other things on the cluster related to CEL and we shouldn't namesquat it.

Reply:
sure, updated it